Library/Homebrew/formula.rb

```diff
@@ -1021,6 +1021,7 @@ def run_post_install
       TMPDIR:        HOMEBREW_TEMP,
       TEMP:          HOMEBREW_TEMP,
       TMP:           HOMEBREW_TEMP,
+      _JAVA_OPTIONS: "#{ENV["_JAVA_OPTIONS"]&.+(" ")}-Djava.io.tmpdir=#{HOMEBREW_TEMP}",
```

Comment:
When is `ENV["_JAVA_OPTIONS"]` set?

Reply:
I don't know actually - the test env has been doing something similar for years.

Reply:
Maybe a legacy of Jenkins pre-env filtering? I'd say remove them both for now and we can change if it breaks.